ArduPlane/quadplane.cpp

```diff
@@ -550,7 +581,8 @@ const AP_Param::ConversionInfo mot_pwm_conversion_table[] = {
 };
 
 QuadPlane::QuadPlane(AP_AHRS &_ahrs) :
-    ahrs(_ahrs)
+    ahrs(_ahrs),
+    command_model_pilot(PILOT_Y_RATE_DEFAULT, PILOT_Y_EXPO_DEFAULT, 0.0)
```

Comment:
worth considering a bit of expo here if it really does make quadplanes nicer to fly